index.bs

```diff
@@ -168,6 +168,9 @@ interface VRDisplay : EventTarget {
    */
   void submitFrame();
 };
+
+VRDisplay implements EventTarget;
+VRDisplay implements VRDisplayEventHandlers;
```

Comment:
Is this the direction everyone is heading? We preferred partials over implements UNLESS multiple things would use the implements. At this time, only VR would use one set of handlers and only VRDisplay would use the other set. So why implements instead of partial or simply adidng it on the interface directly in this case since we are in a single spec? Curious what others think here.